docs/6. How do I.../19. Check WhatsApp Quality Rating and Messaging Limits.md

```diff
@@ -1,3 +1,39 @@
+## Check Whatsapp Messaging limits
+
+Messaging limits determine the maximum number of business-initiated conversations you can initiate using each of your phone numbers in a rolling 24-hour period. A business-initiated conversation begins when the first template message is delivered to a customer and ends 24 hours later.
+
+The phone number tier of your WhatsApp API account will determine your messaging limit - the number of Contacts an account can message. There tiers a phone number inside a WABA can reach, including:
+
+
+_**Tier 1**_: Send template messages to 1K Contacts in a rolling 24-hour period.
+
+_**Tier 2**_: Send template messages to 10K unique Contacts in a rolling 24-hour period
+
+_**Tier 3**_: Send template messages to 100K Contacts in a rolling 24-hour period
+
+_**Tier 4**_: Send template messages to unlimited unique Contacts in a rolling 24-hour period
+
+
+
+
```

Comment:
I think a line should be added that after 500 business conversations in 7days, user would be moved to next tier before giving examples directly